js/index.js

```diff
@@ -40,3 +40,102 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// Navigation
+let navAnchor = document.querySelectorAll('a');
+navAnchor[0].textContent = siteContent.nav["nav-item-1"];
+navAnchor[1].textContent = siteContent.nav["nav-item-2"];
+navAnchor[2].textContent = siteContent.nav["nav-item-3"];
+navAnchor[3].textContent = siteContent.nav["nav-item-4"];
+navAnchor[4].textContent = siteContent.nav["nav-item-5"];
+navAnchor[5].textContent = siteContent.nav["nav-item-6"];
+
+// Add more elements to the navigation items
+let nav = document.querySelector('nav');
+
+let homeElement = document.createElement('a');
+homeElement.text = 'Home';
+homeElement.href = 'index.html';
+homeElement.style.color = 'green';
+
+let projectElement = document.createElement('a');
+projectElement.text = 'Projects';
+projectElement.href = '#';
+projectElement.style.color = 'green';
+
+nav.prepend(homeElement);
+nav.appendChild(projectElement);
+
+// Make nav anchor links to be color green
+let navAnchorArray = Array.from(navAnchor)
+navAnchorArray.forEach(navItem => navItem.style.color = 'green');
```

Comment:
When you use querySelectorAll, it returns an array like element. This means you don't need to extract an array from it, but you could use forEach directly on your navAnchor.